apps/aecore/lib/aecore/oracle/tx/oracle_query_tx.ex

```diff
@@ -263,17 +261,13 @@ defmodule Aecore.Oracle.Tx.OracleQueryTx do
     tx_fee_is_met =
       case query_ttl do
         %{ttl: ttl, type: :relative} ->
-          fee >= calculate_minimum_fee(ttl)
+          fee - GovernanceConstants.oracle_query_base_fee() >= Oracle.calculate_minimum_fee(ttl)
```

Comment:
why is this minus the base fee now?

Reply:
maybe split the minus into a named variable before the greater equals check

Reply:
same applies to other parts of the code

Reply:
I could check the type in Oracle.calculate_minimum_fee and have the base fee there but I think it's cleaner to have it like so (less lines) and as far as moving it to a variable, it would be something like fee_left_for_ttl or something silly like that, I think it's fine like this

Reply:
Its just good style to split maths and comparisons